apiserver/plane/utils/issue_filters.py

```
@@ -562,32 +562,31 @@
-            if item != "null"
-        ]
-
-        # query = Q()
-        # for row in custom_properties:
-        #     key, value = row.split(":")
-        #     query &= Q(
-        #         Q(custom_properties__project_custom_property_id=key) &
-        #         Q(custom_properties__value=value)
-        #     )
+        raw_props = params.get("custom_properties", "")
         
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)